packages/kyt-core/src/config/webpackHotDevClient.js

```diff
@@ -0,0 +1,273 @@
+// This alternative WebpackDevServer combines the functionality of:
+// https://github.com/webpack/webpack-dev-server/blob/webpack-1/client/index.js
+// https://github.com/webpack/webpack/blob/webpack-1/hot/dev-server.js
+
+// It only supports their simplest configuration (hot updates on same server).
+// It makes some opinionated choices on top, like adding a syntax error overlay
+// that looks similar to our console output. The error overlay is inspired by:
+// https://github.com/glenjamin/webpack-hot-middleware
+
+/* eslint-disable no-use-before-define */
```

Comment:
I presume this can't be avoided here

Reply:
it cannot, this is mostly a copy of: https://github.com/jaredpalmer/razzle/blob/master/packages/razzle-dev-utils/webpackHotDevClient.js